### PR TITLE
Made different serializers for Pep list and detail

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/serializers/company_serializers.py [hunk 1]
```diff
@@ -65,11 +65,16 @@ def get_founder_of_count(self, company):
     class Meta:
         model = Company
         fields = (
-            'id', 'name', 'short_name', 'company_type',
-            'edrpou', 'status', 'founder_of_count', 'is_closed',
+            'id', 'name', 'company_type', 'edrpou', 'status', 'founder_of_count', 'is_closed',
```
Comment (roman-tiukh) [inline business_register/serializers/company_serializers.py:68]: i need short_name here.

File: business_register/serializers/company_serializers.py [hunk 2]
```diff
@@ -65,11 +65,16 @@ def get_founder_of_count(self, company):
...
         )
 
 
+class CompanyShortSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Company
+        fields = ('name', 'edrpou',)
```
Comment (roman-tiukh) [inline business_register/serializers/company_serializers.py:75]: id

File: business_register/serializers/pep_serializers.py [hunk 3]
```diff
@@ -60,3 +68,20 @@ class Meta:
                   'termination_date', 'reason_of_termination', 'reason_of_termination_eng',
                   'related_persons', 'related_companies', 'check_companies',
                   )
+
+
+class PepListSerializer(serializers.ModelSerializer):
+    related_persons = RelatedPersonSerializer(many=True)
+    related_companies = CompanyLinkWithPepSerializer(many=True)
```
Comment (roman-tiukh) [inline business_register/serializers/pep_serializers.py:75]: i offer to remove this fields from list serializer

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/serializers/company_serializers.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_serializers.py:68]: restored

File: business_register/serializers/company_serializers.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_serializers.py:75]: added, but, please, explain, why we have to provide our internal ID in API?

File: business_register/serializers/pep_serializers.py (hunk 3 above)
Comment (IegorSoboliev) [reply business_register/serializers/pep_serializers.py:75]: no - we agreed with Liuba to provide in a list all info that we receive from ANTAC. No more, no less)

File: business_register/serializers/company_serializers.py (hunk 2 above)
Comment (roman-tiukh) [reply business_register/serializers/company_serializers.py:75]: for query full info about the company
/company/id/

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**